hazelcast/src/main/java/com/hazelcast/spi/impl/BasicOperationService.java

```diff
@@ -1090,6 +1101,8 @@ private void assertNoBackupOnPrimaryMember(InternalPartition partition, Address
 
         public static final int DELAY_MILLIS = 1000;
 
+        private volatile boolean shutdown;
```

Comment:
we are interrupting thread, so is it really needed ```shutdown``` field. how about ``ìsInterrupted()``` ?